CMake/ExternalData.cmake

```diff
@@ -415,19 +405,18 @@ function(ExternalData_add_target target)
     list(GET tuple 1 name)
     if(NOT DEFINED "_ExternalData_FILE_${file}")
       set("_ExternalData_FILE_${file}" 1)
-      get_property(added DIRECTORY PROPERTY "_ExternalData_FILE_${file}")
+      get_property(
+        added
+        DIRECTORY
+        PROPERTY "_ExternalData_FILE_${file}")
       if(NOT added)
         set_property(DIRECTORY PROPERTY "_ExternalData_FILE_${file}" 1)
         add_custom_command(
           COMMENT "Generating ${file}"
           OUTPUT "${file}"
-          COMMAND ${CMAKE_COMMAND} -Drelative_top=${CMAKE_BINARY_DIR}
-                                   -Dfile=${file} -Dname=${name}
-                                   -DExternalData_ACTION=local
-                                   -DExternalData_CONFIG=${config}
-                                   -P ${_ExternalData_SELF}
-          MAIN_DEPENDENCY "${name}"
-          )
+          COMMAND ${CMAKE_COMMAND} -Drelative_top=${CMAKE_BINARY_DIR} -Dfile=${file} -Dname=${name}
```

Comment:
This was more readable previously. Can this be configured?

Reply:
I tweaked the configuration. Also, I excluded this file, which is tracked externally.